demos/bootstrap.js

```diff
@@ -0,0 +1,54 @@
+(function() {
+
+// Find the script element
+var scripts = document.getElementsByTagName( "script" );
+var script = scripts[ scripts.length - 1 ];
+
+// Read the modules
+var modules = script.getAttribute( "data-modules" );
+var pathParts = window.location.pathname.split( "/" );
+var effectsAll = [
+	"effect-blind",
+	"effect-bounce",
+	"effect-clip",
+	"effect-drop",
+	"effect-explode",
+	"effect-fade",
+	"effect-fold",
+	"effect-highlight",
+	"effect-puff",
+	"effect-pulsate",
+	"effect-scale",
+	"effect-shake",
+	"effect-size",
+	"effect-slide"
+];
```

Comment:
If we added a new effect it would also throw an error when you tried to run it if it was missing from this list.